src/namer/namer.ts

```diff
@@ -79,3 +84,50 @@ function cloneLanguageInfo(graph: any) {
     }
   }
 }
+
+let acronyms = [
+  "Acl",
+  "Api",
+  "Ascii",
+  "Cpu",
+  "Css",
+  "Dns",
+  "Eof",
+  "Guid",
+  "Html",
+  "Http",
+  "Https",
+  "Id",
+  "Ip",
+  "Json",
+  "Lhs",
+  "Qps",
+  "Ram",
+  "Rfc", // TODO check
+  "Rhs",
+  "Rpc",
+  "Sla",
+  "Smtp",
+  "Sql",
+  "Ssh",
+  "Tcp",
+  "Tls",
+  "Ttl",
+  "Udp",
+  "Ui",
+  "Uid",
+  "Uuid",
+  "Uri",
+  "Url",
+  "Utf8",
+  "Vm",
+  "Xml",
+  "Xsrf",
+  "Xss"]
+
+function capitalizeAcronyms(name: string): string {
+  for (const word of acronyms) {
+    name = name.replace(word, word.toUpperCase())
```

Comment:
I don't think this will do a case-insensitive comparison.  If we didn't do that in track 1 then this is probably ok although we might need to revisit later.

Reply:
Yes, that's right. This wont do that. But I didn't want to implement that just yet, since I'm thinking through scenarios where it could cause problems. 
i.e. (a very unlikely example) curl would end up being cURL if I lowercase everything.

Reply:
Good point, that would look bad (in fact I want to say I've seen this in track 1 codegen).